src/Fantomas.Tests/LetBindingTests.fs

```diff
@@ -277,4 +277,36 @@ let ``should keep space before :`` () =
     formatSourceString false "let refl<'a> : Teq<'a, 'a> = Teq(id,   id)" config
     |> fun formatted -> formatSourceString false formatted config
     |> should equal "let refl<'a> : Teq<'a, 'a> = Teq(id, id)
-"
\ No newline at end of file
+"
+
+[<Test>]
+let ``NewLine trivia before simple sequence doesn't force remaining to get offset by last expression column index`` () =
```

Comment:
So I have two nitpicks here, 
* Lowercase the first word
* Place the GitHub link above the test.